android-exoplayer/src/main/java/com/previewseekbar/PreviewSeekBarLayout.java

```diff
@@ -0,0 +1,97 @@
+package com.previewseekbar;
+
+import android.content.Context;
+import android.os.Build;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.FrameLayout;
+
+import com.previewseekbar.base.PreviewGeneralLayout;
+import com.previewseekbar.base.PreviewView;
+
+public class PreviewSeekBarLayout extends PreviewGeneralLayout {
+
+    private PreviewSeekBar seekBar;
+    private FrameLayout previewFrameLayout;
+
+    public PreviewSeekBarLayout(Context context) {
+        super(context);
+    }
+
+    public PreviewSeekBarLayout(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public PreviewSeekBarLayout(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+    }
+
+    @Override
+    public PreviewView getPreviewView() {
+        return seekBar;
+    }
+
+    @Override
+    public FrameLayout getPreviewFrameLayout() {
+        return previewFrameLayout;
+    }
+
+    /**
+     * Align seekbar thumb with the frame layout center
+     */
+    @Override
+    public void setupMargins() {
+        // TODO: Enable the following when adding the thumbnails
```

Comment:
Should this still be commented?

Reply:
Yes, because the streams currently do not have the thumbnails for different duration. I have created https://imggaming.atlassian.net/browse/STR-1015 ticket after which we could enable the progress thumbnails feature.